src/lib/tsgen/runner.ts

```diff
@@ -26,7 +26,7 @@ function createOutputPath(outputFile: string) {
   return outputPath
 }
 
-export default async function tsgenRunner(outputFile: string, contentTypes: any[], prefix = '', includeDocumentation = true) {
+export default async function tsgenRunner(outputFile: string, contentTypes: any[], prefix = '', includeDocumentation = true, systemFields: boolean = false) {
```

Comment:
Hi @aman19K , I've added this so that the regular functioning of the plugin doesn't get affected.
And because I need to change some internal data based on if `systemFields` is true or not.

Reply:
done